MailCoreUI/Utils/FloatingPanelHelper.swift

```diff
@@ -119,6 +122,16 @@ public struct SelfSizingPanelBackportViewModifier: ViewModifier {
                 }
             }
         }
+        .overlay(alignment: .topTrailing) {
+            Button {
+                dismiss()
+            } label: {
+                Image(systemName: "xmark")
+                    .foregroundColor(.accentColor)
+            }
+            .padding(.top, IKPadding.medium)
+            .padding(.trailing, IKPadding.medium)
```

Comment:
```suggestion
            .padding([.top, .trailing], value: .medium)
```
You can use a single padding modifier. You should also use the `padding(_, value:)` modifier with a `IKPadding`